osf_tests/factories.py

```diff
@@ -257,10 +257,14 @@ class InstitutionFactory(DjangoModelFactory):
     email_domains = FakeList('domain_name', n=1)
     orcid_record_verified_source = ''
     delegation_protocol = ''
+    can_request_access = False
```

Comment:
I would default this to true, just because it's going to true going forward, but I'm not sure if it will break a bunch of tests consider using True here.

Reply:
I would leave this as False, please. We don't want Product to have to turn this off for every institution to start with. We can always change it to True after launch once it goes fully into use.